### PR TITLE
A user can select and see the correct answer

### PR DESCRIPTION
**Pull request by thisisseyma**

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -30,6 +30,22 @@ export const initQuestionPage = () => {
...
+      const buttonColor = document.getElementById(`${key}`);
+      if(key == currentQuestion.correct){
+        buttonColor.style.backgroundColor = 'green';
+      }
+      else{ 
+        buttonColor.style.backgroundColor = 'red';
+        const correctAnswer = document.getElementById(currentQuestion.correct);
+        correctAnswer.style.backgroundColor = 'green';
+      };
+
```
Comment (hadeel43) [inline src/pages/questionPage.js:44]: Looks good, also could be:
`      buttonColor.style.backgroundColor =
        key == currentQuestion.correct ? 'green' : 'red';
      const correctAnswer = document.getElementById(currentQuestion.correct);
      correctAnswer.style.backgroundColor = 'green'; `

File: src/views/answerView.js [hunk 2]
```diff
@@ -4,8 +4,11 @@
  */
 export const createAnswerElement = (key, answerText) => {
   const element = document.createElement('li');
-  element.innerHTML = String.raw`
-    ${key}: ${answerText};
-  `;
+  const optionButton = document.createElement('button');
+  optionButton.type = 'button';
+  optionButton.id = `${key}`;
```
Comment (hadeel43) [inline src/views/answerView.js:9]: same here, we don't need to use the backticks and {}:
`optionButton.id = key;`

**Review by hadeel43 — CHANGES_REQUESTED** [review]

File: src/views/answerView.js (hunk 2 above)
Comment (thisisseyma) [reply src/views/answerView.js:9]: Thank you I've changed this one also!

File: src/pages/questionPage.js [hunk 3]
```diff
@@ -30,6 +30,22 @@ export const initQuestionPage = () => {
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+
+    answerElement.addEventListener('click', () => {
+      const buttonColor = document.getElementById(`${key}`);
```
[the remark this answers is not in the document]
Comment (thisisseyma) [reply src/pages/questionPage.js:35]: Thank you I've changed.

File: src/pages/questionPage.js (hunk 1 above)
Comment (thisisseyma) [reply src/pages/questionPage.js:44]: Thanks Hadeel! May I keep the existing one because I get really confused with '? :' structure

**Review by hadeel43 — APPROVED** [review]

**Pull request merged**